skyfield/positionlib.py

```diff
@@ -157,6 +157,43 @@ def separation_from(self, another_icrf):
         c = dots(u1, u2)
         return Angle(radians=arccos(clip(c, -1.0, 1.0)))
 
+    def cirs_xyz(self, epoch):
+        """Compute cartesian CIRS coordinates at a given epoch (x, y, z).
+
+        Calculate coordinates in the Celestial Intermediate Reference System
+        (CIRS), a dynamical coordinate system referenced to the Celestial
+        Intermediate Origin (CIO). As this is a dynamical system it must be
+        calculated at a specific epoch.
+        """
+        if epoch is not None:
```

Comment:
What if the opposite is true — what if `epoch` is indeed `None`?

Reply:
(Oh — and, I have a vague feeling I should really have named this parameter `equinox` this whole time. What do you think? Does that sound more correct? Should we go ahead and name it that here?

Reply:
Good point, should have thought about this more carefully. I've made it throw the same `ValueError` if it gets `None`.

As for the parameter name ,`epoch` is probably the correct terminology.

Reply:
My worry is this wikipedia article (which could of course be half incorrect):

"Equinox is often confused with epoch, with the difference between the two being that the equinox addresses changes in the coordinate system, while the epoch addresses changes in the position of the celestial body itself"

https://en.wikipedia.org/wiki/Equinox_(celestial_coordinates)

But it's okay for now to simply make this match the rest of the API.

Reply:
Hmmmm. I've never come across this distinction before. That nomenclature might make sense for traditional equinox based coordinates, though probably maintaining consistency for the moment is more important.

For CIO based coordinates (e.g. the CIRS) using `equinox` might be very confusing as it's a coordinate system designed to be separated from the actual equinoxes!